inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/RecommendationService.java

```diff
@@ -50,6 +51,10 @@
     
     Recommender getRecommender(long aId);
 
+    Optional<Recommender> getRecommender(Project aProject, String aName);
+
+    boolean existsRecommender(Project aProject, String aName);
```

Comment:
Why not `recommenderExists`?

Reply:
E.g in projectService it's also called `existsProject()` - following existing naming convention.

Reply:
I guess if this were to be changed a better name could be "hasXXX()" or "containsXXX()". or simply.